t/plugin/sls-logger.t

```diff
@@ -188,9 +184,41 @@ hello world
             ngx.say(data)
         }
     }
---- request
-GET /t
 --- response_body
 123
---- no_error_log
-[error]
+
+
+
+=== TEST 7: sls log get milliseconds
+--- config
+    location /t {
+        content_by_lua_block {
+            local function get_syslog_timestamp_millisecond(log_entry)
+                local first_idx = string.find(log_entry, " ") + 1
+                local last_idx2 = string.find(log_entry, " ", first_idx)
+                local rfc3339_date = string.sub(log_entry, first_idx, last_idx2)
+                local rfc3339_len = string.len(rfc3339_date)
+                local rfc3339_millisecond = string.sub(rfc3339_date, rfc3339_len - 4, rfc3339_len - 2)
+                return tonumber(rfc3339_millisecond)
+            end
+
+            math.randomseed(os.time())
+            local rfc5424 = require("apisix.plugins.slslog.rfc5424")
+            local m = 0
+            for i = 1, 5 do
+                ngx.sleep(string.format("%0.3f", math.random()))
```

Comment:
Why use `for i = 1, 5 do` and sleep randomly?

Reply:
Because `ngx.now` may have a millisecond value of `0`, perform multiple tests to ensure that the final result meets expectations.

Reply:
Let's add this explanation as a comment.

Reply:
done